frontend/src/app/components/Chrome/Sidebar/NavLinks.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
+import { Link } from "react-router-dom";
+
+export default class NavLinks extends Component {
+    public state = {
+        id: 'userID',
+        isLoggedIn: false,
+        moduleText: ''
+    };
+
+    public render() {
+        if (!this.state.isLoggedIn) {
+            fetch("/system/sling/info.sessionInfo.json")
```

Comment:
I'm sure this works and it's fine for now, but I'm betting we make this fetch call once in Brand and again here. It would be cleaner if we could just make the call once and share the data somehow.

Reply:
Investigating how to do this.